lib/DDGC/Schema/Result/User.pm

```diff
@@ -4,6 +4,8 @@ package DDGC::Schema::Result::User;
 
 use Moo;
 extends 'DDGC::Schema::Result';
+with 'DDGC::Schema::Role::Result::User::Subscription';
+
```

Comment:
> I can see these Result classes become column defs and a bunch of with specifications, but
> there comes a point when too many files is as rough to deal with as long files.

To clarify, I just had one `with` statement, not a bunch of `with` statements.  I put all the subs in that one role.  So the dir structure might look like this:
```
.
├── Result
│   ├── Account.pm        # schema definition
│   ├── User.pm             # schema definition
│   └── Role
│        ├── Account.pm   # methods for Result::Account
│        └── User.pm        # methods for Result::User
├── ResultSet
│   ├── Account.pm
│   └── User.pm
...
```
That way I always knew to look for the schema definitions in the `Result/` dir and to look for the methods for Result classes in `Result/Role` dir.  Maybe thats still too many files for you.  Whatever floats your boat!